client/src/components/Overview/ImageGalleryThumbnailColumn.jsx

```diff
@@ -27,18 +27,22 @@ class ImageGalleryThumbnailColumn extends React.Component {
   }
 
   render() {
+    let index = 0;
     let thumbnails = this.props.productStyles.results[0].photos.map((style) => {
-      return <ImageGalleryThumbnail key={style.thumbnail_url} thumbnail_url={style.thumbnail_url} />
+      index += 1;
+      return <ImageGalleryThumbnail key={style.thumbnail_url} thumbnail_url={style.thumbnail_url} index={index - 1} gallery={this.props.gallery} />
     });
 
     return (
       <div className="ig-thumbnail-column" data-testid="ig-tc">
         <ImageGalleryThumbnailColumnUpArrow />
-          <div className="ig-thumbnail-inner-container">
-            <div className="igtc-sliding-inner-div"  style={{ top: IGTC_THUMBNAIL_HEIGHT * -this.state.thumbnailPosition + "px" }}>
-              {thumbnails}
-            </div>
+        <div className="one-percent-high-shim"></div>
```

Comment:
Instead of an element that is only used for spacing, is this something that could be refactored into the style of the `ig-thumbnail-inner-container`? Same for the ImageGalleryThumbnailColumnDownArrow, can you just style that in CSS?

Reply:
We want the HTML to represent the content and the CSS to represent the styling as much as possible, for separation of concerns. At least that has always been my philosophy